doc/contributor/refinements.md

```diff
@@ -24,12 +24,11 @@ module M
 end
 ```
```

Comment:
Could you add your `Example#0` from the PR description to help define R/M/C here?
I think it's better than the current example.